src/main/java/seedu/address/logic/parser/ArgumentTokenizer.java

```diff
@@ -82,7 +82,7 @@ private static int findPrefixPosition(String argsString, String prefix, int from
      *
      * @param argsString      Arguments string of the form: {@code preamble <prefix>value <prefix>value ...}
      * @param prefixPositions Zero-based positions of all prefixes in {@code argsString}
-     * @return                ArgumentMultimap object that maps prefixes to their arguments
+     * @return ArgumentMultimap object that maps prefixes to their arguments
```

Comment:
Did you mean to change this? I think here, they were trying to align the descriptions